informer/src/informer_service/informer_service_handler.rs

```diff
@@ -98,3 +121,68 @@ impl InformerService {
         }
     }
 }
+
+pub struct BlockCreatedEventGenerator {
+    client: Arc<Client>,
+    rate_limiter: Limiter,
+    send_status: ColdEventSendStatus,
+}
+
+impl BlockCreatedEventGenerator {
+    pub fn new(client: Arc<Client>) -> Self {
+        Self {
+            client,
+            rate_limiter: ratelimit::Builder::new().quantum(1).interval(Duration::new(0, 5000)).build(),
+
+            send_status: ColdEventSendStatus::InProgress,
+        }
+    }
+
+    pub fn run(mut self, connection: Connection, mut favorite_block_number: u64) {
+        println!("I am here 3");
+        let rt = Runtime::new().unwrap();
+        rt.spawn(async move {
+            task::spawn(async move {
+                loop {
+                    let event = self.gen(favorite_block_number);
+                    connection.cold_notification(&event);
+                    self.rate_limiter.wait();
+                    match self.send_status {
+                        ColdEventSendStatus::AllEventsSent => {
```

Comment:
Should we need `AllEventSent` for the "BlockGenerated" event?
Block will be generated forever. We ca nsend blocks forever to the client.

Reply:
`AllEventsSent` is necessary for the time that there is no new block un our case. If we want to inform the client continuously, I need to think of a procedure.

Reply:
We should give new block information forever.
For example, the best block is 5 and a client wants to receive new blocks from block number 3.
Foundry should give blocks 3, 4, 5 from disk. Foundry should give blocks 6, 7, 8, ... whenever blocks are generated.